readme.md

```diff
@@ -1,2 +1,7 @@
 # manubot.org webpage source code
 
+## Acknowledgements
+
+Cross-browser tested with BrowserStack:
+
+[![BrowserStack](BrowserStack-logo.png?raw=true)](https://www.browserstack.com)
```

Comment:
Want to try putting this on the same line and then I think it will be an in-line image that will look better?